netlify.toml

```diff
@@ -0,0 +1,6 @@
+[build]
+  command = "npm run buildLatest"
+
+[[redirects]]
+  from = "/"
+  to = "https://build.fluidproject.org/"
```

Comment:
I'm assuming this would cause the deploy preview to redirect to the build site. Is that correct? If so, I think it would still take away from the usefulness of the build preview. I can see how this would make sense for the deploy from main though.

Reply:
Oh, of course. Okay. I'm going to add some code to generate a static HTML file in the `buildManifest` script. Will update the PR later or tomorrow morning with that functionality.

Reply:
Added index generation in bf420ba.